ragstack/colbertbase/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "colbertbase"
```

Comment:
We said we would not create a separate package atm ?

Reply:
yes exactly, we don't need another pyyproject for now, let's add the dependencies in the main ragstack pyproject. Note that CassIO is already available so there's no need add `cassandra-driver`

Reply:
refactored